build/buildPythonSdkByVersion.sh

```diff
@@ -150,12 +150,13 @@ getPythonGpgByVersion() {
 
 
 echo
-echo "Building python 3.10 or newer from source code..."
+# TODO: Determine if we need to continue building newer versions of Python from scratch
+echo "Building python 3.11 or newer from source code..."
 
 getPythonGpgByVersion "/tmp/versionsToBuild.txt" $version
 IFS='.' read -ra SPLIT_VERSION <<< "$PYTHON_VERSION"
 
-if  [ "${SPLIT_VERSION[0]}" == "3" ] && [ "${SPLIT_VERSION[1]}" -ge "10" ]
+if  [ "${SPLIT_VERSION[0]}" == "3" ] && [ "${SPLIT_VERSION[1]}" -ge "11" ]
```

Comment:
What about python 3.11 version, will it also have ODBC library issue as I'm currently working on it, is there any way we could test if the ODBC library issue will occur or not?

Reply:
@qianz2 sorry for the late reply -- I added a new sample app that has the `pyodbc` library as a dependency and can reproduce the error if the platform version in the runtime cannot handle it, so I'm optimistic you should be able to reuse it with your 3.11 changes to verify if the issue will be solved.

Reply:
Thanks, I'll use it for testing on python 3.11